dependency_manager/src/edm_tool/bazel.py

```diff
@@ -0,0 +1,55 @@
+"Bazel related functions for edm_tool."
+import sys
+
+import yaml
+
+
+def generate_deps(args):
+    "Parse the dependencies.yaml and print content of *.bzl file to stdout."
+    with open(args.dependencies_yaml, 'r', encoding='utf-8') as f:
+        deps = yaml.safe_load(f)
+    print("Buid files: ", args.build_file, file=sys.stderr)
```

Comment:
For code consistency in edm I would prefer the use of `logging` here.

Reply:
Thanks. I think I'll remove it.